dev-docs/modules/userid-submodules/lockraim.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: userid
+title: lockr AIM
+description: lockr Alternative Identity Manager sub-module
+useridmodule: lockrAIMIdSystem
+---
+
+Alternative Identity Manager (AIM) is a unified container for identity and data management.
+With AIM’s self-service platform, publishers seamlessly integrate and activate alternative IDs like LiveRamp’s Authenticated Traffic Solution (ATS), Unified ID 2.0 (UID2), ID5 and more. The burden of due diligence and maintenance, coupled with the benefits of server-side calls result in the adoption of multiple alternative IDs, clean rooms like InfoSum and CDPs like Blueconic based on their specific needs.
+
+### **Account Creation | AIM**
+
+Sign up for an [Identity lockr account.](https://sso.loc.kr/console/signup)
+Setup your app and activate the AIM library.
+Compile Prebid with the appropriate configurations, and deploy.
+
+### **Configuration | AIM**
+
+First, make sure to add the lockr’s AIM submodule to your Prebid.js package with:
```

Comment:
This looks like maybe you were going to supply an example `gulp build` call but forgot it?